lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java

```diff
@@ -634,7 +634,7 @@ public Set<String> getAllPlatformsForVersionLessFeature(String featureName, Stri
         }
         return featureNames.stream()
                 .map(f -> getAllPlatformsForFeature(f, libertyVersion, libertyRuntime, requestDelay, documentURI))
-                .flatMap(Set::stream).collect(Collectors.toSet());
+                .flatMap(Set::stream).map(String::toLowerCase).collect(Collectors.toSet());
```

Comment:
so the fix isn't this easy because we should preserve case (for doing completion) and ignore case for doing matching/validation

Reply:
yes this method is only for matching and validation. This method is used to fetch all platforms for a versionless feature
For platform completion and platform diagnostic, we use different method

Reply:
interesting...I would not have thought we need different methods (except for the upper case vs lower case nuance...which could be handled by a boolean flag parameter). Which method is used for completion?

Reply:
I see this is specific to platforms for a specific versionless feature.